libraries/ui/src/Card.tsx

```diff
@@ -10,15 +10,14 @@ type CardProps = {
   imageSrc?: string;
   subtitle?: string;
   ctaUrl?: string;
+  ctaText?: string;
+  isEntireCardClickable?: boolean;
   isExternalUrl?: boolean;
   className?: string;
   imageClassName?: string;
   subtitleClassName?: string;
   children?: React.ReactNode;
-} & (
-  | { ctaText: string; isEntireCardClickable?: false }
-  | { ctaText?: string; isEntireCardClickable: true }
-);
+};
```

Comment:
After adding support for non-link cards (with `ctaUrl` undefined), I decided this clever type was more trouble than it was worth to maintain